epilepsy12/forms_folder/case_form.py

```diff
@@ -124,6 +178,11 @@ def clean_date_of_birth(self):
 
     def clean_nhs_number(self):
         # remove spaces
+        organisation = Organisation.objects.get(id=self.organisation_id)
```

Comment:
What use case does this cover?

Reply:
The form sets the none values in the model to empty strings. This might be because blank is True, so possibly my fault. If the child is from Jersey the URN is cleaned and the NHS number is set to none. I am happy to have some better approach. The clean_* methods are called before the final clean method, so the intention was to do some basic validation here first.